src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -1,9 +1,5 @@
 package lotto.domain
 
 class Lottos(val lottos: List<Lotto>) {
-
-    val size: Int
-        get() {
-            return lottos.size
-        }
+    val size: Int = lottos.size
```

Comment:
지금의 코드와 이전 코드와 같이 get()을 지정하는 것과는 실제 컴파일한 코드가 달라요! 
지금과 같이 코드를 작성하시게되면, Lottos는 size라는 프로퍼티(= 상태)를 가지는 객체가 되버려요. 
```suggestion
    val size: Int
        get() = lottos.size
```

아래는 실제로  Kotlin 두 케이스에 대해서 Java 코드로 Decomfile을 해본 결과입니다. 😃👀

<img width="731" alt="image" src="https://github.com/next-step/kotlin-lotto/assets/47850258/b5004d52-ff37-4814-97e5-153f1496e302">

<img width="816" alt="image" src="https://github.com/next-step/kotlin-lotto/assets/47850258/d5a0180f-6643-44d2-bd53-d77017309068">

Reply:
오옷. 신기하네요!